examples/7SegmentLED_BasicUsage_ChangeOrderDigits

```diff
@@ -0,0 +1,96 @@
+// 
+// Copyright (c) 2019 smdn (https://smdn.jp/)
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#include <Arduino.h>
+#include <TM1637_7SegmentLEDController.h>
+
+#if ESP32
+#define PIN_DIO 16 // The pin number connected to TM1637's DIO (#17)
+#define PIN_CLK 17 // The pin number connected to TM1637's CLK (#18)
+#else
+#define PIN_DIO 6 // The pin number connected to TM1637's DIO (#17)
+#define PIN_CLK 7 // The pin number connected to TM1637's CLK (#18)
+#endif
+
+auto display = TM1637_7SegmentLEDController<6 /* The number of digits of 7 segment display */>(
+  PIN_DIO,
+  PIN_CLK,
+   {2, 1, 0, 5, 4, 3}// change order of digits, default is  {0, 1, 2, 3, 4, 5}
+);
+
+// equivalent to the expression above
+/*
+auto display = TM1637_6Digit7SegmentLEDController(
+  PIN_DIO,
+  PIN_CLK
+);
```

Comment:
+ Please add a same line above

Reply:
What do you mean add a same line above?

Reply:
Ah... I mean add `{2, 1, 0, 5, 4, 3}` as same as above.

Reply:
It seem this have not been fixed yet.
OK, I will fix this later after merging.